src/app/core/services/status.service.ts

```
@@ -3,29 +3,23 @@ import { ApiService } from './api.service';
 
   find(objectType: string, objectId: string): Observable<ExtendedStatus[]> {
-    return forkJoin([
-      this.apiService.get<ExtendedStatus[]>(`/${objectType}/${objectId}/estatuses`),
-      this.orgUserSettingsService.get(),
-    ]).pipe(
-      map(([estatuses, orgUserSettings]) => {
-        const timezone = orgUserSettings?.locale?.timezone || 'UTC';
-
-        return estatuses?.map((estatus) => {
-          estatus.userTimezone = timezone;
+    return this.apiService.get('/' + objectType + '/' + objectId + '/estatuses').pipe(
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)